api/routes/order.js

```diff
@@ -0,0 +1,9 @@
+const router = require('express').Router();
+const order = require('../controllers/order');
+
+router.get('/order/:id', order.ordersForUser);
```

Comment:
When you add all orders routes to /orders, here you can just say: 
```suggestion
router.get('/:id', order.ordersForUser);
```